installation-bundle/src/Resources/config/services.yml

```diff
@@ -11,6 +11,9 @@ services:
             - '@Contao\CoreBundle\Migration\MigrationCollection'
         public: true
 
+    contao.install_tool_twig_extension:
```

Comment:
Are we sure we want to add a service for this that we have to maintain? Or should we rather use a compiler pass and add the extension (like the twig extension does with this implicitly tagged service)?

Reply:
The service already existed since Contao 4.3, so technically we still need to support it now.